make/ocm.mk

```
@@ -73,3 +74,12 @@ ocm/cluster/upgrade:
 .PHONY: ocm/help
 ocm/help:
 	@${OCM_SH} -h
+
+.PHONY: ocm/provision_sts_cluster
+ocm/provision_sts_cluster:
+	@${STS_SH} provision_sts_cluster
+
+.PHONY: ocm/delete_sts_cluster
+ocm/delete_sts_cluster:
```

```suggestion
ocm/sts/cluster/delete:
```
To follow https://github.com/integr8ly/delorean/blob/61a8eafdd7546e2247fe6fefe75a88afb75c2021/make/ocm.mk#L53